.travis.yml

```diff
@@ -14,9 +14,10 @@ env:
     - secure: "AfE+PLANOT0mgUKpgGUu4yIrSNKxoNST5zFBeajONEEE0wl/bZVE6iGqucNPR0EaIgMfk+NGgCLLM7VFfy+HF5alBiqUUOapef+fc91iHVzoBgtlLXOO7E1UntrieBQCodURTaBtotCMe7jtlkNajkvUD6atKTYh1AYJ3JY2bHA="
     - GLIUM_HEADLESS_TESTS=1
   matrix:
+    - FEATURES='' TEST=0
     - FEATURES='headless' TEST=1
```

Comment:
Wouldn't it be better to make the tests that rely on glutin only work if glutin is enabled? I had forgotten that would be needed, but that shouldn't be too hard.